### DISCUSSION TITLE
Way to modify the Y position (elevation from tracked floor)

### DISCUSSION THREAD

**Discussion by vortice3D** [google/model-viewer#4442] [Q&A]
Hi there;

For a development we're carrying out at the moment, we need to be able to modify the Y position (elevation from floor) of the 3d model.

This said, my first thought was implementing It by means of a simple gesture, the way AR Quick Look does right off the box, tapping with two fingers and moving then vertically on screen, but sadly It doesn't work that way with model-viewer (MV).

So I thought about a different (responsive) approach only for Android users, showing a UI control (editbox and/or slider) to able the user set the "elevation" of the model. Problem here is that I've just reviewed the examples and docs, but couldn't find any API related with this functionality (at first time I thought It could be the "camera-target" but that is doing a different thing).

Any help with this?

**Comment by elalish** [comment]
We've talked about this feature in the past, but haven't implemented it yet. PRs welcome! I'd be happy to discuss API shape here if you're interested.
